tests/junit_common.py

```diff
@@ -213,11 +213,12 @@ def parse_test_line(line: str) -> TestSpec:
     raw_test_args: str = args[0][args[0].index('TESTARGS(') + 9:args[0].rindex(')')]
     # transform 'name="myname",only="serial,int32"' into {'name': 'myname', 'only': 'serial,int32'}
     test_args: dict = dict([''.join(t).split('=') for t in re.findall(r"""([^,=]+)(=)"([^"]*)\"""", raw_test_args)])
+    name: str = test_args.get('name', '')
```

Comment:
Do we want to allow for anonymous tests? I guess for unit tests that makes sense.

Reply:
Yeah, most of our tests have no name. I added this option because I found myself repeating very similar names over and over again for the vector state value and access lock tests and those names weren't adding much value.

Reply:
Cool. Makes sense.